packages/models/src/eservice-template/eserviceTemplate.ts

```diff
@@ -37,13 +37,13 @@ export const EServiceTemplateVersion = z.object({
   deprecatedAt: z.coerce.date().optional(),
   attributes: EServiceAttributes,
 
-  // The following fields are values to be set in all the e-service instances of this template, not editable by the user
+  // The following fields are values to be set in all the eservice instances descriptors of this template, not editable by the user
   description: z.string().optional(),
   interface: Document.optional(),
   docs: z.array(Document),
   voucherLifespan: z.number().int(),
 
```

Comment:
Also attributes goes in this group, not in the above one
```suggestion
  voucherLifespan: z.number().int(),
  attributes: EServiceAttributes,
```